src/Sylius/Bundle/ApiBundle/Command/Payment/UpdatePaymentRequest.php

```diff
@@ -31,12 +31,12 @@ public function getPayload(): mixed
         return $this->payload;
     }
 
-    public function getHash(): ?string
+    public function getHash(): string
```

Comment:
Property is still nullable, either `getter` needs to become nullable or some other change to happen.

```php
protected ?string $hash = null;
```

Reply:
It's becouse the hash is being set during the process, so at the start it's null, on the other hand I removed the null return types, as I wanted to get rid of some unnecessary asserts